.github/workflows/macos-dispatch-build-bottle.yml

```diff
@@ -1,4 +1,4 @@
-name: Dispatch build bottle
+name: Dispatch build bottle (macOS)
```

Comment:
I think the name here should stay as-is until we have merged Linux fully in here.

Reply:
Should we also not rename the workflow file for now?